modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -73,7 +73,6 @@ CV__DNN_INLINE_NS_BEGIN
         //! OpenCV is built with Intel OpenVINO or
         //! DNN_BACKEND_OPENCV otherwise.
         DNN_BACKEND_DEFAULT = 0,
-        DNN_BACKEND_HALIDE,
```

Comment:
That probably breaks enum numeration

Reply:
Good point. It also breaks compatibility in case people use enum values directrly instead of enums. Let me discuss with other folks.

Reply:
I propose to remove it in 5.x, but do not touch 4.x to presume API compatibility.

Reply:
@asmorkalov, Just to clarify, do you mean just a enum value.